draft-ietf-mls-protocol.md

```diff
@@ -1573,9 +1568,9 @@ computed as follows:
 
 ~~~~~
 struct {
-    PreSharedKeyID id;
-    uint16 index;
-    uint16 count;
+    PreSharedKeyID psk_id;
+    uint16 psk_index;       // i
```

Comment:
Could you elaborate what hat are "i" and "n" supposed to mean here? Or are those just typos?